src/meta/play-meta.js

```diff
@@ -102,17 +103,29 @@ export const plays = [
     blog: 'https://blog.greenroots.info/how-to-create-react-form-with-a-single-change-event-handler',
     video: ''
   }, {
-    id: 'pl-random-meme-generator', 
-    name: 'Random Meme Generator', 
+    id: 'pl-random-meme-generator',
+    name: 'Random Meme Generator',
     description: 'A project to demonstrate the use of API to fetch random memes! It also demonstrates how you can do event handling!',
-    component: () => {return <RandomMemeGenerator />},
+    component: () => { return <RandomMemeGenerator /> },
     path: '/plays/random-meme-generator',
     level: 'Beginner',
     tags: 'JSX,Hooks,API,EventHandling',
     github: 'murtuzaalisurti',
     cover: '',
     blog: '',
     video: ''
+  }, {
+    id: 'pl-react-todo-app',
+    name: 'React Todo App',
+    description: 'It is a simple Todo App which keeps track of your regular work',
+    component: () => { return <ReactTodoApp /> },
+    path: '/plays/react-todo-app',
+    level: 'Beginner',
+    tags: 'ReactHooks, JavaScript, Css, Local Storage',
```

Comment:
If you have removed localstorage functionality, the tag list shouldn't;t have that.